lib/ActivityStream.jsm

```diff
@@ -239,6 +239,10 @@ const PREFS_CONFIG = new Map([
       });
     },
   }],
+  ["discoverystream.endpoints", {
+    title: "Endpoint prefixes (comma-separated) that are allowed to be requested",
+    value: "https://getpocket.cdn.mozilla.net/",
```

Comment:
Has this been run by mathijs to make sure he's not expecting other pocket urls to be allowed?

Reply:
He provided a list of production and dev values https://bugzilla.mozilla.org/show_bug.cgi?id=1521203#c1 and we're only handling the feeds and not images here, so we end up with just the cdn.